docs/configuration.md

```diff
@@ -639,6 +639,20 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.shuffle.registration.timeout</code></td>
+  <td>5000</td>
+  <td>
+    Timeout in milliseconds for registration to the external service.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.shuffle.registration.maxAttempts</code></td>
+  <td>3</td>
+  <td>
+    When we fail to register to the external service, we will retry for maxAttempts times.
```

Comment:
nit:  When we fail to register to the external shuffle service, we will retry for maxAttempts times.